src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -177,6 +177,32 @@ export default class ReimbursementRequestService {
     return updatedReimbursementRequest;
   }
 
+  /**
+   * Soft-deletes the given reimbursement request
+   *
+   * @param requestId the request to be deleted
+   * @param deleter the user deleting the request
```

Comment:
rename deleter to submitter, and in both comments rename request to reimbursement request